errors.go

```diff
@@ -81,6 +81,28 @@ func (err ConfigurationError) Error() string {
 // See https://cwiki.apache.org/confluence/display/KAFKA/A+Guide+To+The+Kafka+Protocol#AGuideToTheKafkaProtocol-ErrorCodes
 type KError int16
 
+// MultiError is used to contain multi error.
+type MultiError struct {
```

Comment:
What's the reason to add this?

Reply:
Currently the delete record request is sent to leader by leader. Thus we may receive multi error response and probably the errors are not the same.  I use MultiError to contain the array of error.